src/main/java/fr/zcraft/quartzlib/components/i18n/translators/gettext/POFile.java

```diff
@@ -174,6 +174,9 @@ public void parse() throws CannotParsePOException {
             if (!tokens.isEmpty()) {
                 analyseEntry(tokens);
             }
+
+            // At the end we compute plural rules
+            pluralForms = new PluralForms(pluralCount, pluralFormScript);
         } catch (IOException e) {
```

Comment:
It is possible to make use of the autoclosable in a try with ressources, reader is only used here.

Reply:
As the reader is built in the constructor, is this really possible?